src/dash-playlist-loader.js

```diff
@@ -306,7 +345,7 @@ export default class DashPlaylistLoader extends EventTarget {
         return doneFn(master, null);
       }
 
-      const bytes = new Uint8Array(request.response);
+      const bytes = toUint8(request.response);
```

Comment:
Since we potentially re-use bytes and they can already be a uin8 at this point, we use the helper from vhs-utils which only creates a new uint8 if needed.